scripts/release/mule/package/linux/amd64/deb/package.sh

```diff
@@ -16,24 +16,21 @@ if [ -z "$OS_TYPE" ] || [ -z "$ARCH" ] || [ -z "$WORKDIR" ]; then
     exit 1
 fi
 
-REPO_DIR="$WORKDIR"
-BRANCH=$("$REPO_DIR/scripts/compute_branch.sh")
-CHANNEL=$("$REPO_DIR/scripts/compute_branch_channel.sh" "$BRANCH")
-BASE="$REPO_DIR/tmp/node_pkgs/$OS_TYPE/$ARCH"
+BRANCH=${BRANCH:-$(git rev-parse --abbrev-ref HEAD)}
```

Comment:
Is this changing the way that the branch is computed, in addition to accepting it as an argument?

Reply:
I'll change that back to reference the script.  Initially, I had the idea to simplify as much as I could and avoid referencing other scripts, but then I forgot to change that back.  I will now.

Thanks for catching that.

Reply:
And yes, it first defaults to BRANCH if it's set.